app/main.py

```diff
@@ -1 +1,22 @@
-# write your code here
+import os
+import shutil
+
+
+def move_file(command: str) -> None:
+    mode, source, destination = command.split()
```

Comment:
The variable names 'mode', 'source', and 'destination' are descriptive and correct, which is good. However, ensure that the command is split correctly and that it contains exactly three parts. Consider adding a check to handle cases where the command does not have the expected format.